src/Tasks/Microsoft.NET.Build.Tasks/ResolveTargetingPackAssets.cs

```diff
@@ -208,13 +299,34 @@ private void AddNetStandardTargetingPackAssets(ITaskItem targetingPack, string t
             string targetingPackDllFolder,
             ITaskItem targetingPack, List<TaskItem> referenceItems, List<TaskItem> analyzerItems)
         {
```

Comment:
Can this be a static method that takes a `FrameworkListDefinition` as an argument?  Then it won't be able to accidentally access data that's not included in the key.